v3/src/models/document/free-tile-row.ts

```diff
@@ -158,6 +158,14 @@ export const FreeTileRow = TileRowModel
     setTileDimensions(tileId: string, dimensions: { width?: number, height?: number }) {
       const freeTileLayout = self.getNode(tileId)
       freeTileLayout?.setSize(dimensions.width, dimensions.height)
+    },
+    setTilePosition(tileId: string, position: { x?: number, y?: number }) {
+      const freeTileLayout = self.getNode(tileId)
+      if (freeTileLayout) {
+        const x = position.x ?? freeTileLayout.x
+        const y = position.y ?? freeTileLayout.y
```

Comment:
FYI, I think you could also do:
```suggestion
        const { x = freeTileLayout.x, y = freeTileLayout.y } = position
```